frameworks/helloworld/src/main/dist/custom_tld.yml

```diff
@@ -0,0 +1,27 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{SERVICE_PRINCIPAL}}
+  user: {{SERVICE_USER}}
+pods:
+  hello:
+    count: 1
+    uris:
+      - {{BOOTSTRAP_URI}}
```

Comment:
Future note: Should we think about adding this URI automatically?

Reply:
Yes

Reply:
True. Not all pods need it. I was letting the lines between pods and the scheduler (where we now always use it) blur a little.